addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -2237,13 +2238,32 @@ export class OdooEditor extends EventTarget {
                 // Check for url after user insert a space so we won't transform an incomplete url.
                 if (
                     ev.data &&
-                    ev.data.includes(' ') &&
+                    ev.data === ' ' &&
```

Comment:
Will it work in mobile with composition ? Maybe it should check if it ends with a space ?

Reply:
Wait so it does not convert urls in mobile ? That's sad.